var/spack/repos/builtin/packages/eckit/package.py

```diff
@@ -16,6 +16,8 @@ class Eckit(CMakePackage):
 
     maintainers = ["skosukhin", "climbfuji"]
 
+    version("1.20.2", sha256="779be536e22f985b00927330d8430794c43f4f48")
+    version("1.20.0", sha256="5ecebc9548eba8df1c3e6f162e8f480257528eab")
```

Comment:
These don't seem to be valid git commit hashes or sha256 sums

Reply:
You are right, they are much shorter but for whatever reason they worked just fine. I'll replace them with a made-up hash of the right length and see what happens. Same for the other occurrences below.